src/Player.js

```diff
@@ -52,29 +64,20 @@ export default class Player {
   doOtherMovement(p) {
```

Comment:
we should combine the two movement functions into one but im saving that for later

Reply:
Smooth and not smooth? I agree

Reply:
doing this later :)